ms2query/results_table.py

```diff
@@ -47,3 +49,29 @@ def add_parent_masses(self, parent_masses, base_nr_mass_similarity):
 
         self.data["mass_similarity"] = base_nr_mass_similarity ** \
             (np.abs(parent_masses - self.parent_mass))
+
+    def preselect_on_ms2deepscore(self):
+        selected_spectrum_ids = list(self.ms2deepscores.nlargest(
+            self.preselection_cut_off).index)
+        self.data["spectrum_ids"] = pd.Series(selected_spectrum_ids)
+        self.data["ms2ds_score"] = \
+            np.array(self.ms2deepscores.loc[selected_spectrum_ids])
+
+    def add_ms2query_meta_score(self,
+                                predictions):
+        """Add MS2Query meta score to data and sort on this score
+
+        Args:
+        ------
+        predictions:
+            An iterable containing the ms2query model meta scores
+        """
+        self.data["ms2query_model_prediction"] = predictions
+        self.data.sort_values(by=["ms2query_model_prediction"],
+                              ascending=False,
+                              inplace=True)
```

Comment:
The sorting is fine I guess. But I think we should add a test for this (maybe build a unit test for `ResultsTable` anyway).
Can be done later as reaction to #97.